field_audit/management/commands/bootstrap.py

```diff
@@ -0,0 +1,130 @@
+from contextlib import contextmanager
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from field_audit.field_audit import get_audited_models
+from field_audit.models import AuditEvent
+
+
+class Command(BaseCommand):
+
+    help = "Create bootstrap AuditEvent records for audited model classes."
+    models = {}
+
+    @classmethod
+    def setup_models(cls):
+        def model_name(model, unique=False):
+            if unique:
+                return f"{model._meta.app_label}.{model.__name__}"
+            return model.__name__
+        for model_class in get_audited_models():
+            name = model_name(model_class)
+            collision = cls.models.setdefault(name, model_class)
+            if collision is not model_class:
+                del cls.models[name]
+                cls.models[model_name(collision, unique=True)] = collision
+                cls.models[model_name(model_class, unique=True)] = model_class
```

Comment:
Would it make sense to assert they are not the same here?
```suggestion
                assert model_name(collision, unique=True) != model_name(model_class, unique=True), model_class
                cls.models[model_name(collision, unique=True)] = collision
                cls.models[model_name(model_class, unique=True)] = model_class
```

Reply:
Good point. Having two models with the same name in the same app should be prohibited by Django. I think the only way this could happen is if the user specified the same model twice on the commandline. I'll write a test for that scenario and make handle this accordingly.

Reply:
Fixed in 12ffea5.